auth/src/main/res/values/config.xml

```diff
@@ -9,6 +9,16 @@
     -->
     <string name="facebook_application_id" translatable="false">CHANGE-ME</string>
 
+    <!--
+     To enable Chrome Custom Tabs for Facebook Login on devices without the Facebook app
```

Comment:
I'm a little unclear here.  If they don't want to enable Chrome Custom Tabs can they do nothing and have it work?  Or will it try to use fbYOUR_APP_ID as the facebook_login_protocol_scheme and break.